template/css/tailwind.min.css

```diff
@@ -1176,6 +1176,12 @@ a:hover {
   text-transform: uppercase;
 }
 
+.markdown :is(ol, ul) {
+  -webkit-padding-start: 40px;
+          padding-inline-start: 40px;
+  list-style-type: disc;
+}
+
```

Comment:
should we have some customizations in `tailwind.min.css`? no other way?

Reply:
@Theiaz ok, I know why it is here, because you locally ran `npm run generate:assets` and then you committed changes to tailwind. Just please remove and we release. In another PR would be awesome somehow fix it, so postcss is actually processing stuff into a new file that is not in the repo, that we can add to gitignore

thoughts?

Reply:
Yeay your're right. `tailwind.min.css` should be generated from ci/cd. I deleted the file.